astropy/table/table.py

```diff
@@ -588,7 +588,8 @@ def _base_repr_(self, html=False):
         tableid = 'table{id}'.format(id=id(self))
         data_lines, outs = self.formatter._pformat_table(self,
             tableid=tableid, html=html, max_width=(-1 if html else None),
-            show_name=True, show_unit=None, show_dtype=True)
+                                                    show_name=True, show_unit=None,
+                                                         show_dtype=True)
```

Comment:
@agroener - I think you might as well reformat this while getting rid of the sphinx error (@bsipocz - thanks for finding it; I always have trouble with the sphinx output, trying to understand errors!).  I'd do
```
data_lines, outs = self.formatter._pformat_table(
    self, tableid=tableid, html=html, max_width=(-1 if html else None),
    show_name=True, show_unit=None, show_dtype=True)
```